types/jasmine/index.d.ts

```diff
@@ -478,7 +478,7 @@ declare namespace jasmine {
          * @param value The value to pretty-print
          * @return The pretty-printed value
          */
-        pp(value: unknown): string;
+        pp: typeof pp;
```

Comment:
Is there a test for this? If there is one already please tell me the line number.